core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -236,6 +244,8 @@ private void parseArguments(final String[] args) {
             .withRequiredArg()
             .ofType(String.class)
             .describedAs("file name");
+        forceOption = optionParser.accepts("force", "Force remove members when long session time out has been configured, " +
```

Comment:
Do we also want to edit the `usage` info on top to mention the force delete option?

Reply:
I think so, updated